contents.json

```diff
@@ -329,6 +329,18 @@
     }
   ],
   "projects": [
+        {
+                "title": "iSH",
+                "category-ids": ["developer", "emulator"],
+                "tags": ["swift"],
```

Comment:
@PseudonymPatel I'm not sure about `swift`, the project is written mostly in `c` + `objc`. Let's use them instead.